### PR TITLE
Combine demos

### PR DESCRIPTION
**Pull request by gjmooney**
This combines the local and the lite versions of the demo to reduce redundancy.

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -47,7 +25,7 @@ <h1>Local Demo</h1>
 
     <iframe
       id="jupyterlab"
-      src="http://localhost:8888"
+      src="%VITE_DEMO_SRC%"
```
Comment (jtpio) [inline demo/index.html:28]: With that still work when deployed to GitHub pages?

When trying the build `github-pages` artifact (built on CI) locally, and starting an HTTP server with `python -m http.server`, it gives the following:

image

Comment (gjmooney) [reply demo/index.html:28]: I don't think it'll work with the python server, it should work if you run `jlpm start:local` in the `demo` directory, that'll start the Vite server and set the env variables. I'll double check the ghpages deployment on my fork.

Comment (gjmooney) [reply demo/index.html:28]: > With that still work when deployed to GitHub pages?

It does not :melting_face:

Comment (gjmooney) [reply demo/index.html:28]: ok, it should be good now, https://gjmooney.github.io/jupyter-iframe-commands/

File: demo/src/main.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      console.log('command', command);
+      console.log('args', args);
+    }
+
+    commandBridge.execute(command, args ? JSON.parse(args) : {});
+  }
+});
+
+document.getElementById('list-commands').addEventListener('click', async () => {
+  const commands = await commandBridge.listCommands();
```
Comment (jtpio) [inline demo/src/main.js:63]: Maybe we could sort the list of commands, so they are easier to browser in the demo?

File: demo/src/main.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
+/* eslint-disable @typescript-eslint/quotes */
+/* eslint-disable no-undef */
+import { CommandBridge } from 'jupyter-iframe-commands-host';
+import Swal from 'sweetalert2';
```
Comment (jtpio) [inline demo/src/main.js:4]: Also wondering if we could even use the native `<dialog>` directly here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog

**Review by jtpio — APPROVED** [review]
Thanks!

**Pull request merged**